### PR TITLE
Fixed events won't be deleted

### PR DESCRIPTION
**Pull request by Fank**
key before: `plugins:calendar:events:pid:[object Object]`   
key now: `plugins:calendar:events:pid:5`

# THIS IS A URGENT issue and need a update very fast!!!

Because we receive reminder for events which won't exists anymore ....

### CODE REVIEW

File: src/lib/event.js [hunk 1]
```diff
@@ -34,13 +34,13 @@ const saveEvent = (event) => {
...
-const deleteEvent = (pid) => {
-  const objectKey = `${listKey}:pid:${pid}`;
+const deleteEvent = (data) => {
+  const objectKey = `${listKey}:pid:${data.post.pid}`;
   return Promise.all([
     sortedSetRemove(listKey, objectKey),
     sortedSetRemove(listByEndKey, objectKey),
     deleteKey(objectKey),
-    removeAllResponses(pid),
+    removeAllResponses(data),
```
Comment (pitaj) [inline src/lib/event.js]: Change this to `removeAllResponses(data.post.pid)` and remove the changes in `responses.js`

**Comment by Fank** [comment]
@pitaj done

**Comment by pitaj** [comment]
Ok thanks

**Comment by Fank** [comment]
Can you please publish an update?

**Comment by pitaj** [comment]
Yes, in the next couple of hours.

**Pull request merged**